src/Microsoft.AspNetCore.Mvc.Razor/exceptions.netcore.json

```diff
@@ -17,13 +17,13 @@
         "OldTypeId": "public interface Microsoft.AspNetCore.Mvc.Razor.Compilation.ICompilationService",
         "OldMemberId": "Microsoft.AspNetCore.Mvc.Razor.Compilation.CompilationResult Compile(Microsoft.AspNetCore.Mvc.Razor.Compilation.RelativeFileInfo fileInfo, System.String compilationContent)",
         "NewTypeId": "public interface Microsoft.AspNetCore.Mvc.Razor.Compilation.ICompilationService",
-        "NewMemberId": "Microsoft.AspNetCore.Mvc.Razor.Compilation.CompilationResult Compile(Microsoft.AspNetCore.Razor.Evolution.RazorCodeDocument codeDocument, Microsoft.AspNetCore.Razor.Evolution.RazorCSharpDocument cSharpDocument)",
+        "NewMemberId": "Microsoft.AspNetCore.Mvc.Razor.Compilation.CompilationResult Compile(Microsoft.AspNetCore.Razor.Language.RazorCodeDocument codeDocument, Microsoft.AspNetCore.Razor.Language.RazorCSharpDocument cSharpDocument)",
```

Comment:
@NTaylorMullen I'm not sure what you mean. What does "this" refer to in your sentence?

E.g. if you're asking about changing existing exceptions, sure go right ahead and adjust them.

If you're asking about `"Kind": "Modification"`, they're not long for this world. We will be thinking in strict "addition" and "removal" terms. (This is fine because API Check doesn't handle modifications correctly anyhow.) But, nobody has to get out in front of that change.